packages/backup-restore-operator/charts/templates/backup.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
To keep the CRDs separated from the actual elements deployed onto the cluster, we should either prepend the YAML names like `crd-<crd-name>.yaml` or move these into `templates/crds/backup.yml`.

Reply:
This won't have any impact on the actual chart installation, is this separation to make the chart structure more clear?

Reply:
> This won't have any impact on the actual chart installation, is this separation to make the chart structure more clear?

Yes

Reply:
I agree with renaming them cause it makes it clear.